Description: Drop --enable-runtime-cxx-hack from the default configuration, to
 fix FTBFS on armel with gcc4.5
Author: Jani Monoses <jani@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gcc-4.5/+bug/686381
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=607987
Bug: http://www.ntg.nl/pipermail/dev-luatex/2010-December/004056.html
---
 build.sh |    1 -
 1 file changed, 1 deletion(-)

--- luatex.orig/build.sh
+++ luatex/build.sh
@@ -152,7 +152,6 @@
 TL_MAKE=$MAKE ../source/configure  $CONFHOST $CONFBUILD  $WARNINGFLAGS\
     --disable-native-texlive-build \
     --with-banner-add='/Debian' \
-    --enable-cxx-runtime-hack \
     --disable-all-pkgs \
     --enable-shared    \
     --disable-largefile \
